src/hooks/usePayAndDowngrade.ts

```diff
@@ -0,0 +1,33 @@
+import {useEffect, useRef} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import Navigation from '@libs/Navigation/Navigation';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+function usePayAndDowngrade(setIsDeleteModalOpen: (value: boolean) => void) {
+    const [isLoadingBill] = useOnyx(ONYXKEYS.IS_LOADING_BILL_WHEN_DOWNGRADE);
+    const [shouldBillWhenDowngrading] = useOnyx(ONYXKEYS.SHOULD_BILL_WHEN_DOWNGRADING);
```

Comment:
@dukenv0307  What does the `SHOULD_BILL_WHEN_DOWNGRADING` key do? I'm not sure about its purpose, as we have it defined here but are not updating its value anywhere.

Reply:
Okay, right now I'm getting false here, so I'll ask Ben to check its value for the test user.